src/PubTypes.lua

```diff
@@ -46,6 +46,23 @@ export type Task =
 	{Destroy: (any) -> ()} |
 	{Task}
 
+-- Types that can be used as attributes
+
+export type Attribute = 
```

Comment:
This type seems to be unused? Although I could be mistaken.

The 'attribute' value type is set to 'any'

see https://github.com/Elttob/Fusion/pull/199/commits/73cafa0fbbd85721eb2504f2d18ed3f572c3925b#diff-7484de93742d78456b515380dea6075867f53dae6fe6d72b7d431da65e4664ffR18

Reply:
Just realised this, will remove!